### ISSUE TITLE
Dotted zeros instead of slashed zeros.

### ISSUE THREAD

**Issue by dumol** [vercel/geist-font#81]
**Font Name (Geist Sans/Geist Mono):**
- [x] Geist Sans
- [x] Geist Mono

**Description of the Issue:**
Slashed zeros are too similar to Scandinavian letter Ø.

**Steps to Reproduce:**
1.  Compare `0` with `Ø` in Geist Mono.
2. Activate slash zero in Geist Sans with `ss09`.
3. Compare the slashed zero with Ø in Geist Sans.

**Expected Behavior:**
For Geist Mono, use a dot inside the zero to have it easy distinguishable from both `Ø` and `O`.
For Geist Sans, add a zero with a dot inside, and use that for the `zero` stylistic set in Geist Sans.

**Screenshots:**
![20240116_13h50m02s_grim](https://github.com/vercel/geist-font/assets/1620933/b18a16cf-04d7-45a8-9ccf-c676d0b0a929)

**Environment (please complete the following information):**
 - OS: All
 - Software: All
 - Version of the Font: 1.002

**Additional Context:**
Dotted zero is a common solution to this problem, as implemented in JetBrains Mono, IBM Plex Mono, Source Code Pro, etc.

**Comment by luciascarlet** [comment]
This is just my opinion, but I don't like the look of dotted zeroes at all and tend to change them to slashed zeroes whenever it's allowed by a font. It's also not a good idea for low-DPI rendering as it often ends up being off-centre.

However, I really like how Atkinson Hyperlegible went about solving this same problem of distinguishing Ø and 0, where the slashed 0 is simply flipped. This will render well on any screen, looks nicer (IMO), and is legible.

<img width="628" alt="image" src="https://github.com/vercel/geist-font/assets/13824526/a1d49e0d-7fc2-4acd-9e08-71b008052684">

**Comment by dumol** [comment]
I agree with the above suggestion of using an inverted slashed zero.

On a tangent, I'm currently using a low-dpi device where I use a collection of fonts and FontConfig rules to improve legibility: https://github.com/dumol/PixelPerfect/. The `ui-monospace` font that I'm setting through those rules is [Luculent](http://eastfarthing.com/luculent/), which actually has an inverted slashed zero. GitHub uses `ui-monospace` for monospaced texts here, that's how I noticed…

(For the record, my personal preference is for a dotted zero, but given the attention Geist is getting for rendering on low-dpi devices, I guess the inverted slashed zero would be a better default, as argued above by @luciascarlet.)

Thanks!
